app/models/katello/erratum.rb

```diff
@@ -118,6 +118,12 @@ def update_from_json(json)
       keys = %w(title id severity issued type description reboot_suggested solution updated summary)
       custom_json = json.slice(*keys)
 
+      # handle SUSE epoch dates
+      if custom_json["issued"].to_i.to_s == custom_json["issued"]
+        custom_json["issued"] = epoch_to_datetime(custom_json["issued"])
+        custom_json["updated"] = epoch_to_datetime(custom_json["updated"]) unless custom_json["updated"].blank?
```

Comment:
I would probably have different checks for issued and updated.  Sure most likely if its an epoch with issued, its going to be an epoch with updated but that may not always be true, so i would handle them separately.